### PR TITLE
Help component

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
Integrate help-component as initial hints with 'First steps' to help understanding the app

### CODE REVIEW

**Comment by thiloSchlemmer** [comment]
> What do you think about renaming the whole component to first steps -> , FirstSteps.js, FIRST_STEPS_CONTENT_URL, etc... ?

Sure

File: src/modules/help/components/FirstSteps.js [hunk 1]
```diff
@@ -47,24 +52,32 @@ export class Survey extends MvuElement {
...
 	 */
 	onInitialize() {
+		const { ConfigService: configService } = $injector.inject('ConfigService');
+
 		this.observe(state => state.media, media => this.signal(Update_IsPortrait_HasMinWidth, { isPortrait: media.portrait, hasMinWidth: media.minWidth }));
 		this.observe(state => state.mainMenu, mainMenu => this.signal(Update_IsOpen_TabIndex, { isOpen: mainMenu.open, tabIndex: mainMenu.tab }));
 
 		this.signal(Update_HasBeenVisible, this._environmentService.getUrlParams().get(QueryParameters.T_DISABLE_INITIAL_UI_HINTS) === 'true');
+
+		if (configService.hasKey('FIRST_STEPS_CONTENT_URL')) {
```
Comment (taulinger) [inline src/modules/help/components/FirstSteps.js:71]: else path not taken

File: src/modules/help/i18n/firstSteps.provider.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+export const firstStepsProvide = (lang) => {
+	switch (lang) {
+
+		case 'en':
+			return {
+				//the first part of the snake_case key should be the name of the related module
```
Comment (taulinger) [inline src/modules/help/i18n/firstSteps.provider.js:6]: Probably we will have more components within the help module in the future. Let's give them keys indicating their
concrete component:
help_button -> help_first-steps_button

File: src/modules/help/i18n/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 
-import { surveyProvide } from './survey.provider';
+import { firstStepsProvide } from './firstSteps.provider';
 import { $injector } from '../../../injection';
 const { TranslationService: translationService } = $injector.inject('TranslationService');
-translationService.register('surveyProvide', surveyProvide);
+translationService.register('initialHintsProvide', firstStepsProvide);
```
Comment (taulinger) [inline src/modules/help/i18n/index.js:5]: `translationService.register('firstStepsProvider', firstStepsProvide);`

File: src/modules/map/components/olMap/handler/measure/OlMeasurementHandler.js [hunk 4]
```diff
@@ -202,11 +202,15 @@ export class OlMeasurementHandler extends OlLayerHandler {
...
-		const pointerUpHandler = () => {
+		const pointerUpHandler = (event) => {
 			const draggingOverlay = getOverlays(this._vectorLayer).find(o => o.get('dragging') === true);
 			if (draggingOverlay) {
 				draggingOverlay.set('dragging', false);
 			}
+			const coordinate = event.coordinate;
+			const dragging = event.dragging;
+			const pixel = event.pixel;
+			this._updateMeasureState(coordinate, pixel, dragging);
```
Comment (taulinger) [inline src/modules/map/components/olMap/handler/measure/OlMeasurementHandler.js:213]: What's the purpose of these changes within the context of this branch?

**Review by taulinger — COMMENTED** [review]
Nice proceedings, would yet suggest some small changes

**Comment by thiloSchlemmer** [comment]
Thx for your hints!

**Review by taulinger — APPROVED** [review]
Nice 👍
